tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -815,21 +825,26 @@ func computeIterations(run *v1alpha1.Run, tls *pipelineloopv1alpha1.PipelineLoop
 	iterationElements := []interface{}{}
 	iterationParamStr := ""
 	iterationParamStrSeparator := ""
+	var err error
 	for _, p := range run.Spec.Params {
 		if p.Name == "from" {
-			from, _ = strconv.Atoi(p.Value.StringVal)
-			fromProvided = true
+			from, err = getIntegerParamValue(p)
```

Comment:
In practice, I have not seen a problem due to "\n" terminated strings passed as a value. My understanding was these are already taken care of by the parsers. 

Can you provide an example ?

Reply:
@ScrapCodes , we have an example, the value of the range is from an upper task result,  e.g: an output of a bash script task.  The result is `3\n` I see.

Reply:
These are `from` and `to` parameters of a loop, do you think they are generated as a result of running a bash script?